src/loading.js

```diff
@@ -1,35 +1,43 @@
+/* eslint-disable react/no-multi-comp */
 import React from 'react';
-import map from 'lodash/map';
+import every from 'lodash/every';
 import some from 'lodash/some';
+import omit from 'lodash/omit';
+import constant from 'lodash/constant';
 import displayName from './displayName';
-import _isReady from './isReady';
 
-const _isLoading = ({ readyState }) => {
-  return some(map(readyState, rs => rs.loading));
-};
+export const defaultIsLoading = ({ readyState }) => some(readyState, 'loading');
+
+export const defaultIsReady = ({ readyState }) => every(readyState, 'ready');
+
+export class NoWrapper extends React.Component { render = () => (this.props.children || [])[0] || null; }
+export class NoLoader extends React.Component { render = () => null; }
 
 export default function loading({
-  isLoading = _isLoading,
-  isReady = _isReady,
-  wrapper = <div />,
-  loader = <div>loading...</div>,
-  loaderProps = () => ({}),
-  wrapperProps = () => ({})
-}) {
+  isLoading = defaultIsLoading,
+  isReady = defaultIsReady,
+  wrapper = <NoWrapper />,
+  loader = <NoLoader />,
+  loaderProps = constant({}),
```

Comment:
Not a big deal for me, but are you sure `() => ({})`, or even `(() => ({}))` is more readable? It's just `lodash` after all http://devdocs.io/lodash~4/index#constant and it does what it 'says': return a constant function